modules/kn-service-describe.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative_serving/serverless-applications.adoc
+// * serverless/cli_reference/kn-services-ref.adoc
+
+[id="kn-service-describe_{context}"]
+= Describing serverless applications
+
+You can describe a Knative service by using the `kn` CLI `kn service describe` command.
+
+.Example commands
+
+* Describe a service:
++
+[source,terminal]
+----
+$ kn service describe <service-name>
+----
++
+.Example output
++
+[source,terminal]
+----
+Name:         example-service
+Namespace:    knative-serving
+Annotations:  serving.knative.openshift.io/ownerName=knative-serving, serving.knative.openshift.i ...
+Age:          2h
+URL:          http://example-service-knative-serving.apps.ocp1.redhat.com
+Revisions:
+  100%  @latest (example-service-00002) [2] (2h)
+        Image:  registry.redhat.io/openshift-serverless-1/example-service-artifacts-rhel8@sha256:e75ffcb0d051f9abb945eaa91ae281858387f92f65c884a49b8801560cbf4965 (at e75ffc)
```

Comment:
```suggestion
        Image:  registry.redhat.io/openshift-serverless-1/example-service-artifacts-rhel8:v1 (at e75ffc)
```

Instead of the sha we could use the tag here as this is easier to format and doesn't lead to horizontal scrolling. `v1` is made up here, but for the context of this example I think it is not important.